service/src/test/java/uk/gov/hmcts/reform/fpl/service/LocalAuthorityUserServiceTest.java

```diff
@@ -91,7 +91,7 @@ void shouldMakeCallToUpdateCaseRoleEndpointWhenUsersWithinLocalAuthority() {
 
         verifyUpdateCaseRolesWasCalledThisManyTimesForEachUser(1, USER_IDS);
 
-        verify(client, times(1)).authenticateUser(anyString(), anyString());
+        verify(client, times(1)).getAccessToken(anyString(), anyString());
```

Comment:
I know that it isn't really part of what you are doing in this PR but do you want to remove the `times(1)`?